### PR TITLE
Feature/87 Roster - create basic view

### PR DESCRIPTION
**Pull request by tjanowsk**
rota

### CODE REVIEW

File: frontend/src/components/rota/rota.module.scss [hunk 1]
```diff
@@ -1,5 +1,16 @@
 .rota {
   grid-area: main;
-  width: 100%;
-  height: 100%;
+  display: grid;
+  height: 100vh;
+  grid-template:
+    'header' min-content
+    'calendar' 1fr;
+  flex-direction: column;
```
Comment (AdrianBorg) [inline frontend/src/components/rota/rota.module.scss:8]: is this relevant for `grid` display?
thought it's just for `flex`

File: frontend/src/components/rota/index.tsx [hunk 2]
```diff
@@ -1,10 +1,58 @@
...
+  {
+    start: '2020-05-08T11:28Z',
+    end: '2020-05-08T13:33Z',
+    title: 'lunch',
+    description: 'FOOOD',
+    color: 'gold',
+  },
+];
+
+const tooltipFunction = (info: any) => {
```
Comment (AdrianBorg) [inline frontend/src/components/rota/index.tsx:28]: is there a way of specifying what happens if the description is empty/undefined?
But actually, should we even worry about it?

Comment (tjanowsk) [reply frontend/src/components/rota/index.tsx:28]: We'll be the ones choosing what to put in the description (I think it's name, surname, times and engagement type?), so I wouldn't worry about it at this stage.

File: frontend/src/components/rota/rota.module.scss (hunk 1 above)
Comment (tjanowsk) [reply frontend/src/components/rota/rota.module.scss:8]: Ah, I was playing around with different layouts and forgot to remove this. Will fix.

File: frontend/public/main.css [hunk 3]
```diff
@@ -0,0 +1,7 @@
+body {
```
Comment (emab) [inline frontend/public/main.css:1]: Did this not work if it was in `index.module.scss`?

File: frontend/src/components/rota/rota.module.scss [hunk 4]
```diff
@@ -1,5 +1,16 @@
...
+  margin: 0 1em;
+  grid-template:
+    'header' min-content
+    'calendar' 1fr;
+  h1 {
+    grid-area: header;
+  }
+}
+
+.calendarContainer {
```
Comment (AdrianBorg) [inline frontend/src/components/rota/rota.module.scss:14]: > I changed the colour of 'today' header to yellow, but I couldn't do this in the rota's scss file, so I had to put it in main.css.
> My guess is that scss is compiled before fullcalendar.io classes are generated, so the styling is not applied to them. I have checked that the styling is not applied at all, it's not overwritten.
> There may be a way around this by e.g. playing with webpack's settings. Suggestions welcome.

would you have a similar result if instead you set the styling here with the global tag?

so something like:
``` 
.calendarContainer {
  grid-area: calendar;
  
  & :global(.fc-today) {
    background-color: yellow;
  }
}
```

Havent tried myself but I think that should make any child class of `calendarContianer` with the name `fc-today` get that styling.

Comment (tjanowsk) [reply frontend/src/components/rota/rota.module.scss:14]: That worked, thanks!

**Pull request merged**